2022-05-metafacture-workshop/index.html

```diff
@@ -0,0 +1,939 @@
+			</section>
+
+			<section>
+				<h1>Dokumentation</h1>
+				<p><span class="fragment">Was für Module gibt es? Was machen die?</span></p>
+				<span class="fragment"><pre><code class="bash" data-trim>
+$ flux.sh # oder flux.bat auf Windows
+				</code></pre></span>
+				<p><span class="fragment"><a href="https://github.com/metafacture/metafacture-documentation/blob/master/flux-commands.md">
+				https://github.com/metafacture/metafacture-documentation/blob/master/flux-commands.md</a></span></p>
+				<p><span class="fragment">z.B. statt JSON als "Formeta" ausgeben, formatiert</span></p>
+			</section>
+
+			<section>
+				<h2>encode-formeta(style="multiline")</h2>
```

Comment:
Würde nicht Formeta, sondern ein übliches Format nehmen, wie YAML oder so. Formeta hat keine relevanz für Bibliotheksnutzer:innen.

Reply:
Aber für Metafacture-Nutzer:innen, weil das schön die interne Struktur abbildet, und auch z.B. in Tripeln genutzt wird. Und, nicht zuletzt, wieder das von oben, weil es Bestandteil der bestehenden Beispiele und sonstigen Materialien ist. Wenn wir from scratch einen Workshop aufsetzen würden wäre es was anderes.

Reply:
Wie wird das in den Triplen genutzt?

Reply:
s. stream-to-triples Abbildung, "Serialised with Formeta", ist mir auch nicht ganz klar wo/wie man das braucht, aber ich dachte da noch, ja, ist gut Formeta drin zu haben.